project/Build.scala

```diff
@@ -4,17 +4,18 @@ import sbt.Keys._
 object Build extends Build {
   import Dependencies._
   import Formatting._
+  import Publishing._
 
   lazy val basicSettings = Seq(
     organization := "com.scalapenos",
-    version := "0.1.0",
+    version := "0.1.1-SNAPSHOT",
     scalaVersion := "2.11.7",
-    crossScalaVersions := Seq("2.11.7", "2.10.5"),
+    crossScalaVersions := Seq("2.11.7", "2.10.6"),
```

Comment:
Yeah, but I'll remove it in one big separate PR. 
This change was just my OCD seeing an outdated version number :smile: